custom_components/renault/binary_sensor.py

```diff
@@ -50,7 +55,8 @@ class RenaultPluggedInSensor(RenaultBatteryDataEntity, BinarySensorEntity):
     @property
     def is_on(self):
         """Return true if the binary sensor is on."""
-        return self.coordinator.data.get("plugStatus") == 1
+        if self.data.plugStatus is not None:  # Zero can be a valid value
+            return self.data.get_plug_status() == PlugState.PLUGGED
```

Comment:
if 0, should the binary be at on, off, undefined or unavailable state ?

Reply:
I think it should be undefined. I will explicit `None`